code/datums/diseases/advance/symptoms/sensory.dm

```diff
@@ -0,0 +1,95 @@
+/*
+//////////////////////////////////////
+Sensory-Restoration
+	Very very very very noticable.
+	Lowers resistance tremendously.
+	Decreases stage speed tremendously.
+	Decreases transmittablity tremendously.
+	Fatal.
+Bonus
+	The body generates Sensory restorational chemicals.
+	inacusiate for ears
+	antihol for removal of alcohol
+	synaptizine to purge sensory hallucigens
+	mannitol to kickstart the mind
+
+//////////////////////////////////////
+*/
+/datum/symptom/sensory_restoration
+	name = "Sensory Restoration"
+	stealth = -1
+	resistance = -4
+	stage_speed = -4
+	transmittable = -3
+	level = 6
+	severity = 0
+
+/datum/symptom/sensory_restoration/Activate(var/datum/disease/advance/A)
+	..()
+	if(prob(SYMPTOM_ACTIVATION_PROB * 3))
+		var/mob/living/M = A.affected_mob
+		switch(A.stage)
+			if(2)
+				if(M.reagents.get_reagent_amount("inacusiate")<10)
+					M.reagents.add_reagent("inacusiate"=10)
+					M << "<span class='notice'><b>Your hearing feels clearer and crisp</b></span>"
```

Comment:
Missing the period at the end
Also the bold text in these looks weird and inconsistent imo